sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatAsyncClient.java

```diff
@@ -317,7 +317,9 @@ public void stopRealtimeNotifications() {
      * @param deviceRegistrationToken Device registration token obtained from the FCM SDK.
      * @param errorHandler error handler callback for registration failures
      * @throws RuntimeException if push notifications failed to start.
+     * @deprecated The function will be replaced by startPushNotifications(String deviceRegistrationToken)
```

Comment:
When pointing users to an alternative to the deprecated API, I think we could benefit from using a JavaDoc link. I'm not 100% sure this is the right syntax but you get the idea.
```suggestion
     * @deprecated The function will be replaced by {@link ChatAsyncClient#startPushNotifications(String)}
```